.vscode/c_cpp_properties.json

```
@@ -0,0 +1,9 @@
+{
+    "configurations": [
+        {
+            "name": "CMake",
+            "configurationProvider": "ms-vscode.cmake-tools"
+        }
+    ],
+    "version": 4
+}
```

nit: missing newline
```suggestion
}

```